### PR TITLE
Explain in the readme what the plugin does.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki 
/cc @lrytz

File: README.md [hunk 1]
```diff
@@ -28,6 +28,46 @@ libraryDependencies ++= {
...
+    assertEquals(5, 2 + 3)
+}
+
+class MyTests extends MyTestsBase
+```
+
+Scala 2.12.0-M5 identifies that `add()` does not need a forward in `MyTests`,
+because it will be inherited as a default method on the JVM.
+But this fools JUnit 4, which does not look for `@Test` methods in interfaces.
+In this case, the plugin requires `MyTests` as follows:
```
Comment (lrytz) [inline README.md]: maybe "changes" / "updates" instead of "requires"?

**Comment by lrytz** [comment]
nice, thanks!

File: README.md (hunk 1 above)
Comment (sjrd) [inline README.md]: Er that was meant to be "rewrites" ... No idea how that became "requires" between my brain and my fingers ...

**Comment by nicolasstucki** [comment]
LGTM

**Pull request merged**